portals/devportal/source/src/app/components/Apis/Details/GraphQLConsole/GraphQLUI.jsx

```diff
@@ -94,13 +95,19 @@ export default function GraphQLUI(props) {
         } else {
             token = 'Bearer ' + accessTokenProvider();
         }
+        const headers = {
+            Accept: 'application/json',
+            'Content-Type': 'application/json',
+            [authorizationHeader]: token,
+        };
+
+        additionalHeaders.forEach((header) => {
```

Comment:
You can simply spread the `additionalHeaders` object in to above  `headers` object when creating the `headers` variable

Reply:
```js
        const headers = {
            Accept: 'application/json',
            'Content-Type': 'application/json',
            [authorizationHeader]: token,
            ...additionalHeaders
        };
```

Reply:
We will have drop the id from additionalHeaders to pass as per the current implementation